portfolio/src/main/webapp/style.css

```diff
@@ -1,14 +1,21 @@
 #content {
```

Comment:
Here's what I see on a smaller screen, FYI: ![image](https://user-images.githubusercontent.com/2783959/83071876-a8d3b380-a022-11ea-85e3-b348026981d4.png)

Not saying it needs to be addressed in this PR, but the navigation section overlaps the main content when the browser is narrower.

Reply:
I noticed this problem too and I think I will find a way to fix the problem in the next PR